charts/ccsm-helm/charts/zeebe-gateway/templates/_helpers.tpl

```diff
@@ -0,0 +1,47 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "zeebe-gateway.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s-gateway" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "zeebe.version" -}}
+{{- printf "%s:%s" .Values.global.image.repository .Values.global.image.tag -}}
+{{- end -}}
+
+{{- define "zeebe.labels.gateway" -}}
+{{- template "zeebe.labels" . }}
+app.kubernetes.io/component: gateway
```

Comment:
```suggestion
app.kubernetes.io/component: zeebe-gateway
```
Would this be more consistent?

Reply:
Not sure, for the brokers we have as label, `component: broker` :shrug: Would you prefer it ? :)

Reply:
I guess if we not look from a zeebe focused perspective but from a CCSM user perspective, I would expect prefixed labels with the component names, i.e. `zeebe-`, `operate-` etc.

So yes personally I would expect `zeebe-gateway` and `zeebe-broker`

Reply:
Makes sense to me let me do that as part of https://github.com/camunda-community-hub/camunda-cloud-helm/pull/147